Dashboard/Views/StatusView.axaml

```diff
@@ -1,8 +1,246 @@
+                                    </Grid.ColumnDefinitions>
+                                    <TextBlock Text="Emergency Brake Service Status:" FontSize="24" Grid.Column="0" Margin="0,0,10,10" VerticalAlignment="Center"/>
+                                    <TextBlock Text="Ready" FontSize="22" Grid.Column="1" VerticalAlignment="Center" Background="White" Padding="10,0" TextAlignment="Center"/>
+                                </Grid>
+                                <!-- Steering Angle actual -->
+                                <TextBlock Text="Steering Angle (º Degrees)" FontSize="24" />
+                                <Grid>
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="Auto" />
+                                        <ColumnDefinition Width="Auto" />
+                                    </Grid.ColumnDefinitions>
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition Height="Auto" />
+                                        <RowDefinition Height="Auto" />
+                                    </Grid.RowDefinitions>
+                                    <Slider Grid.Row="0" Minimum="-100" Maximum="100" Value="13" Width="800" IsEnabled="False" Grid.Column="0" Margin="0,0,20,0"/>
+                                    <TextBlock Text="Actual" VerticalAlignment="Center" Grid.Column="0" Grid.Row="1"/>
+                                    <TextBlock Grid.Row="0" Text="13º" FontSize="22" VerticalAlignment="Center" Background="White" Padding="20,0" TextAlignment="Center" Grid.Column="1"/>
+                                </Grid>
+                                <!-- Steering Angle target -->
+                                <Grid>
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="Auto" />
+                                        <ColumnDefinition Width="Auto" />
+                                    </Grid.ColumnDefinitions>
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition Height="Auto" />
+                                        <RowDefinition Height="Auto" />
+                                    </Grid.RowDefinitions>
+                                    <Slider Grid.Row="0" Minimum="-100" Maximum="100" Value="17" Width="800" IsEnabled="False" Grid.Column="0" Margin="0,0,20,0"/>
+                                    <TextBlock Text="Target" VerticalAlignment="Center" Grid.Column="0" Grid.Row="1"/>
+                                    <TextBlock Grid.Row="0" Text="17º" FontSize="22" VerticalAlignment="Center" Background="White" Padding="20,0" TextAlignment="Center" Grid.Column="1"/>
+                                </Grid>
+                                <Grid HorizontalAlignment="Left" VerticalAlignment="Top" Margin="0,0,0,0">
```

Comment:
Same q about alignment, can we make the actual and target (for speed and brake) value backgrounds aligned please